src/NzbDrone.Core/MediaFiles/EpisodeFileMovingService.cs

```diff
@@ -134,7 +134,15 @@ private EpisodeFile TransferFile(EpisodeFile episodeFile, Series series, List<Ep
                     case ScriptImportDecision.DeferMove:
                         break;
                     case ScriptImportDecision.RenameRequested:
-                        MoveEpisodeFile(episodeFile, series, episodeFile.Episodes);
+                        try
```

Comment:
This switch case should be converted to an if/else block instead of moving more logic inside of it.

Reply:
Done